src/InboundServer/api.yaml

```diff
@@ -2643,6 +2738,26 @@ components:
         see API Signature document.
       schema:
         type: string
+    traceparent:
+      name: traceparent
+      in: header
+      required: false
+      description: >-
+        The traceparent header field is a non-HTTP standard field used by the API
+        for passing along the parent traceId for children traces to use to create child
+        spans.
+      schema:
+        type: string
+    tracestate:
+      name: tracestate
+      in: header
+      required: false
+      description: >-
+        The tracestate header field is a non-HTTP standard field used by the API
+        for passing along the current state of the parent traceId being passed in
+        the traceparent header
+      schema:
+        type: string
```

Comment:
See pattern/schema based on the WC3 standard.

Reply:
pattern added